catalog/catalog.yaml

```diff
@@ -79,6 +79,27 @@ components:
           x-go-type-skip-optional-pointer: true
 
     Media:
+      properties:
+        dark:
```

Comment:
Can you add some descriptions here to specify what "dark" and "light" is? Is dark the darker version of the logo? Or is it the version of the logo to be used for dark mode? I'm personally unsure. Or even clearer would be to name these `default` and `darkMode` I think. 

Also I think one of them should be optional (if a logo does't have a specific version for dark mode, then we can default to the regular version)